src/main/java/services/RestoreFactory.java

```diff
@@ -338,6 +338,9 @@ private String getSyncToken(String username) {
         if (username == null) {
             return null;
         }
+
+        username = UserUtils.unwrapUsername(username);
```

Comment:
Well when its passed in its potentially *not* unwrapped so I guess its going to be inconsistent at one point in the code